### PR TITLE
Team/delete

### PR DESCRIPTION
**Pull request by GilmarPozzo**

### CODE REVIEW

File: src/test/java/com/webaut/project/steps/TeamSteps.java [hunk 1]
```diff
@@ -3,27 +3,35 @@
...
     private Home home;
     private TeamForm teamForm;
     private TeamDetails teamDetails;
     private Header header;
     private HeaderCreateMenu headerCreateMenuHeader;
+    private TeamSideBarOption teamSideBarOption;
+    private TeamDeleteConfirmation teamDeleteConfirmation;
 
-    public TeamSteps (Home home, TeamForm teamForm, TeamDetails teamDetails, Header header, HeaderCreateMenu headerCreateMenuHeader){
+    public TeamSteps (Home home, TeamForm teamForm, TeamDetails teamDetails, Header header, HeaderCreateMenu headerCreateMenuHeader, TeamSideBarOption teamSideBarOption, TeamDeleteConfirmation teamDeleteConfirmation){
```
Comment (lizzymendivil) [inline src/test/java/com/webaut/project/steps/TeamSteps.java]: This code line is too long, maybe you should shorten it

**Review by lizzymendivil — CHANGES_REQUESTED** [review]
Did you run `gradle clean check` ?

**Comment by lizzymendivil** [comment]
Code review done

File: src/test/java/com/webaut/project/steps/TeamSteps.java (hunk 1 above)
Comment (GilmarPozzo) [reply src/test/java/com/webaut/project/steps/TeamSteps.java]: It is the price of Injection, I've talked Carlos About, and there is no problem, if you don´t have any another concern please, Merge it.

Comment (lizzymendivil) [reply src/test/java/com/webaut/project/steps/TeamSteps.java]: Did you run gradle clean check ?

Comment (carledriss) [reply src/test/java/com/webaut/project/steps/TeamSteps.java]: the gradle clean check is not enabled on this project.
please skip that in the meanwhile.

Comment (lizzymendivil) [reply src/test/java/com/webaut/project/steps/TeamSteps.java]: In my case, this command is working fine.

Comment (lufercc) [reply src/test/java/com/webaut/project/steps/TeamSteps.java]: JIC enabled Travis and fix all TC failed
please verify Travis is a success before merge this

**Pull request merged**